land_use/models.py

```diff
@@ -0,0 +1,247 @@
+from django.contrib.gis.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class LandUseTI(models.Model):
+    """LandUseTI model data for land_use model."""
+    id = models.IntegerField(
```

Comment:
Missing blank spaces between attributes.

Reply:
Removed all missing blank in models's attributes.